src/lib/config.ts

```diff
@@ -239,6 +239,7 @@ export class default_config {
         x: "stop",
         gi: "focusinput -l",
         "g?": "rot13",
+        "g!": "jumble",
```

Comment:
Can we add an ex alias for `reibadailty`? I liked that joke

Reply:
Added